CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Changelog
 
+## 0.4.5
+
+### Ruff's language server is now in Beta
+
+`v0.4.5` marks the official Beta release of `ruff server`, an integrated language server built into Ruff.
+`ruff server` does everything `ruff-lsp` does and more, bringing significant improvements to performance along the way.
```

Comment:
Can we saw something like:

> `ruff server` supports the same feature set as `ruff-lsp`, but with superior performance and no installation required. We'd love your feedback.

Just to match the language we added to the https://github.com/astral-sh/ruff-lsp README and elsewhere.

Reply:
Lets add a second paragraph that's just like:

You can enable `ruff server` in the [VS Code extension](https://github.com/astral-sh/ruff-vscode?tab=readme-ov-file#enabling-the-rust-based-language-server) today.

What do you think?

Reply:
Should we mention that `ruff server` has several brand-new features too?

Reply:
I just want there to be a call-to-action without going through the blog post.

Reply:
Actually:

> `ruff server` supports the same feature set as `ruff-lsp`, powering linting, formatting, and code fixes in Ruff's editor integrations -- but with superior performance and no installation required. We'd love your feedback.

Something to make clear it's not a general LSP?

Reply:
Ah, to be clear, this was a question for the first suggestion.

Reply:
@charliermarsh That sounds good!